### PR TITLE
ES-855596 UG for Console application in DocIo and Presentation

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
UG for Console application in DocIo adn Presentation.
Task Link: <a href="https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/855548">Blur the version of NuGet Package image in both DocIO and Presentation</a>
Regards
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/Create-Word-document-in-Console-application.md [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
+
+# Create a Word document in Console Application
+
+Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **create a Word document in Console Application**.
+
+## Create a Word document using .NET Core and Latest
+
+The below steps illustrates creating a simple **Word document** in console application using **.NET Core**.
+
+Step 1: Create a new **Console application** project.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Create-Word-document-in-Console-application.md:17]: Mention what type of console (.NET Framework or .NET Core console).
Add all the screenshots of Visual Studio to identify what framework you choosing for project

File: File-Formats/DocIO/Create-Word-document-in-Console-application.md [hunk 2]
```diff
@@ -0,0 +1,533 @@
...
+
+Click [here](https://www.syncfusion.com/document-processing/word-framework/net-core) to explore the rich set of Syncfusion Word library (DocIO) features. 
+
+An online sample link to [create a Word document](https://ej2.syncfusion.com/aspnetcore/Word/HelloWorld#/material3) in ASP.NET Core. 
+
+## Create Word document in .NET Framework
+
+The below steps illustrates creating a simple **Word document** in console application using **.NET Framework**.
+
+Step 1: Create a new **Console application** project.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Create-Word-document-in-Console-application.md:283]: Address the same feedback above shared.

File: File-Formats/DocIO/Create-Word-document-in-Console-application.md [hunk 3]
```diff
@@ -0,0 +1,533 @@
...
+![Create a C# Console application in Visual Studio](Console-Images/.NET-FrameWork/Console-Template-Net-FrameWork.png)
+
+Step 2: Install [Syncfusion.DocIO.WinForms](https://www.nuget.org/packages/Syncfusion.DocIO.WinForms/) NuGet package as a reference to your Windows Forms application from the [NuGet.org](https://www.nuget.org/).
+
+![Install Syncfusion.DocIO.WinForms NuGet package](Console-Images/.NET-FrameWork/Nuget-Package-NET-FrameWork.png)
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering Syncfusion license key in your application to use our components.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Create-Word-document-in-Console-application.md:290]: Add the Note about WinForms NuGet as 1st point.
2nd point - you can add license related note

File: File-Formats/DocIO/Create-Word-document-in-Console-application.md [hunk 4]
```diff
@@ -0,0 +1,533 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+You can download a complete working sample from GitHub.
+
+By executing the program, you will get the **Word document** as follows.
+
+![.NET FrameWork output Word document](Mac-images/GettingStartedOutput.jpg)
+
+Click [here](https://www.syncfusion.com/document-processing/word-framework/net) to explore the rich set of Syncfusion Word library (DocIO) features.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Create-Word-document-in-Console-application.md:530]: Remove these demo and online pages link

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Create-Word-document-in-Console-application.md [hunk 5]
```diff
@@ -0,0 +1,533 @@
...
+//Append paragraph.
+section.AddParagraph();
+
+//Save the Word document
+document.Save("Sample.docx");
+
+{% endhighlight %}
+{% endtabs %}
+
+You can download a complete working sample from GitHub.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Create-Word-document-in-Console-application.md:524]: Github link?

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Create-Word-document-in-Console-application.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Create-Word-document-in-Console-application.md:17]: Modified

File: File-Formats/DocIO/Create-Word-document-in-Console-application.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Create-Word-document-in-Console-application.md:283]: Modified

File: File-Formats/DocIO/Create-Word-document-in-Console-application.md (hunk 4 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Create-Word-document-in-Console-application.md:530]: modified

File: File-Formats/DocIO/Create-Word-document-in-Console-application.md (hunk 3 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Create-Word-document-in-Console-application.md:290]: Added

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**